### PR TITLE
1.2.0 Changelog

### PR DESCRIPTION
**Pull request by Kukks**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
...
 
+## 1.2.0
+
+### Improvements:
+* Migrate to Bootstrap5 (#2490) @dennisreimann
+* Greenfield: Server Info: Support all currency codes for sync status (#2511) @kukks
+* Greenfield: Add StoreId to Invoice model (#2592)  @kukks
+* Greenfield: Change `enabledOnly` filter to `enabled` @kukks
+* Self host PoS app default images (#2449) @dennisreimann
+* Varius UI Tweaks and improvements (#2558 #2562 #2568 #2572 #2606 #2608 #2615 #2627 #2628 #2649 #2645 #2673 #2646 #2647) @dstrukt @dennisreimann @woutersamaey @johanf85 @bolatovumar
```
Comment (dennisreimann) [inline Changelog.md:11]:
```suggestion
* Various UI Tweaks and improvements (#2558 #2562 #2568 #2572 #2606 #2608 #2615 #2627 #2628 #2649 #2645 #2673 #2646 #2647) @dstrukt @dennisreimann @woutersamaey @johanf85 @bolatovumar
```

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,55 @@
...
+* Notify users to use newer BTCPay Vault app if necessary. @nicolasdorier
```
Comment (dennisreimann) [inline Changelog.md:12]:
```suggestion
* Notify users to use newer BTCPay Vault app if necessary @nicolasdorier
```

File: Changelog.md [hunk 3]
```diff
@@ -1,5 +1,55 @@
...
+* Set lightning invoice fallback in QR code as uppercase (#2492) @bjarnemagnussen @Kukks
+* Optimize payout database fetching @nicolasdorier
+* Wallet Signing UI improvements (#2559) @dennisreimann
+* Add payjoin to hot wallet setup and turn on by default @dennisreimann
```
Comment (dennisreimann) [inline Changelog.md:16]:
```suggestion
* Add payjoin to hot wallet setup and turn on by default (#2450) @dennisreimann
```

File: Changelog.md [hunk 4]
```diff
@@ -1,5 +1,55 @@
...
+* Add webhook delivery status indicator (#2679) @bolatovumar
+* Auto-select store when creating a new invoice (#2680) @bolatovumar
+* Save paymentRequestId in Metadata when creating invoice for Payment Request (#2644) @woutersamaey
+* Support multiple file upload (#2705) @cypherbeerus
+
+### New features:
+* Greenfield: Delete User API (#2340) @bolatovumar @kukks
+* Greenfield: Add misc/permissions to document the hierarchical structure (#2654) @nicolasdorier
+* Greenfield: Add "skip" and "limit" params for onchain txs API endpoint (#2688) @bolatovumar
+* Greenfield: Add CanModifyInvoices Permissions (#2595) @kukks
```
Comment (dennisreimann) [inline Changelog.md:30]:
```suggestion
* Greenfield: Add `CanModifyInvoices` permission (#2595) @kukks
```

File: Changelog.md [hunk 5]
```diff
@@ -1,5 +1,55 @@
...
+* Greenfield: Delete User API (#2340) @bolatovumar @kukks
+* Greenfield: Add misc/permissions to document the hierarchical structure (#2654) @nicolasdorier
+* Greenfield: Add "skip" and "limit" params for onchain txs API endpoint (#2688) @bolatovumar
+* Greenfield: Add `CanModifyInvoices` permission (#2595) @kukks
+* Greenfield: Add text search terms to an invoice (#2648) @NicolasDorier
+* Greenfield: Add Get store Payment methods API (#2545) @kukks @bolatovumar
+* GreenField: Add Generate Store OnChain Wallet API (#2708) @kukks
+* Test Webhooks functionality (#2474) @bolatovumar
+* Allow marking payout as paid manually (#2539) @Kukks
+* Payouts: Detect External OnChain Payouts (#2462) @Kukks
```
Comment (NicolasDorier) [inline Changelog.md:36]:
```suggestion
* Pull payments: Detect External OnChain Payouts (#2462) @Kukks
```

**Comment by NicolasDorier** [comment]
#2727

**Comment by dennisreimann** [comment]
@NicolasDorier correct, the changes were made in #2575 which was merged after the latest release. 👍

**Comment by NicolasDorier** [comment]
Need to add #2730 merging soon.

File: Changelog.md [hunk 6]
```diff
@@ -1,5 +1,55 @@
...
+* Introduce Server paging for Payouts List (#2564) @kukks @dennisreimann
+* Hide referer URL to hide our BTCPay Server URL (#2655) @woutersamaey
+* Deeper accessibility for plugin system @kukks
+* Add webhook delivery status indicator (#2679) @bolatovumar
+* Auto-select store when creating a new invoice (#2680) @bolatovumar
+* Save paymentRequestId in Metadata when creating invoice for Payment Request (#2644) @woutersamaey
+* Support multiple file upload (#2705) @cypherbeerus
+
+### New features:
+* Greenfield: Delete User API (#2340) @bolatovumar @kukks
```
Comment (NicolasDorier) [inline Changelog.md:27]:
```suggestion
* Can create invoices without a specific amount: Top-up invoices (#2730 #2659) @NicolasDorier
* Greenfield: Delete User API (#2340) @bolatovumar @kukks
```

File: Changelog.md [hunk 7]
```diff
@@ -1,5 +1,55 @@
...
 
+## 1.2.0
+
+### Improvements:
+* Migrate to Bootstrap5 (#2490) @dennisreimann
+* Greenfield: Server Info: Support all currency codes for sync status (#2511) @kukks
+* Greenfield: Add StoreId to Invoice model (#2592)  @kukks
+* Greenfield: Change `enabledOnly` filter to `enabled` @kukks
+* Self host PoS app default images (#2449) @dennisreimann
+* Various UI Tweaks and improvements (#2558 #2562 #2568 #2572 #2606 #2608 #2615 #2627 #2628 #2649 #2645 #2673 #2646 #2647) @dstrukt @dennisreimann @woutersamaey @johanf85 @bolatovumar
```
Comment (pavlenex) [inline Changelog.md:11]:
```suggestion
* Various UI Tweaks and improvements (#2558 #2562 #2568 #2572 #2606 #2608 #2615 #2627 #2628 #2649 #2645 #2673 #2646 #2647 #2745) @dstrukt @dennisreimann @woutersamaey @johanf85 @bolatovumar
```

**Comment by NicolasDorier** [comment]
PoS UI fixes #2744

**Comment by NicolasDorier** [comment]
https://github.com/btcpayserver/btcpayserver/pull/2748

**Comment by NicolasDorier** [comment]
https://github.com/btcpayserver/btcpayserver/pull/2750

**Comment by Zaxounette** [comment]
@NicolasDorier 

`Support spending to Taproot (#2718) @nicolasdorier `

The changelog says the above, but the actual PR title says something else.
Is this correct?
If it is, it's simply adding compatibility to send to Taproot formatted addresses? (Asking for the blog post).

**Comment by NicolasDorier** [comment]
@Zaxounette this is correct. The support of taproot only need an update of NBitcoin, which I did.

**Comment by Zaxounette** [comment]
Thanks!

File: Changelog.md [hunk 8]
```diff
@@ -1,5 +1,55 @@
...
+* Greenfield: Add `CanModifyInvoices` permission (#2595) @kukks
+* Greenfield: Add text search terms to an invoice (#2648) @NicolasDorier
+* Greenfield: Add Get store Payment methods API (#2545) @kukks @bolatovumar
+* GreenField: Add Generate Store OnChain Wallet API (#2708) @kukks
+* Test Webhooks functionality (#2474) @bolatovumar
+* Allow marking payout as paid manually (#2539) @Kukks
+* Pull payments: Detect External OnChain Payouts (#2462) @Kukks
+* Auto-detect language on payment page (#2552) @woutersamaey @Kukks
+* Support spending to Taproot (#2718) @nicolasdorier
+* Show Immature Balance in walletsend page (#2731 @732) @sageprogrammer @nicolasdorier
```
Comment (NicolasDorier) [inline Changelog.md:39]:
```suggestion
* Show Immature Balance in walletsend page (#2731 @732) @sageprogrammer @nicolasdorier
* Add hebrew translation for checkout (https://github.com/btcpayserver/btcpayserver/commit/7ac83575d4c50e42f2ecc02c8bf80f66697b6d57) @jonathanalevi
* Add korean translation for checkout (https://github.com/btcpayserver/btcpayserver/commit/7ac83575d4c50e42f2ecc02c8bf80f66697b6d57)  Saeyoung Kim
```

File: Changelog.md [hunk 9]
```diff
@@ -1,5 +1,55 @@
...
+* Wallet Signing UI improvements (#2559) @dennisreimann
+* Add payjoin to hot wallet setup and turn on by default (#2450) @dennisreimann
+* Add permission code to API page (#2599)  @woutersamaey @dennisreimann
+* Introduce Server paging for Payouts List (#2564) @kukks @dennisreimann
+* Hide referer URL to hide our BTCPay Server URL (#2655) @woutersamaey
+* Deeper accessibility for plugin system @kukks
+* Add webhook delivery status indicator (#2679) @bolatovumar
+* Auto-select store when creating a new invoice (#2680) @bolatovumar
+* Save paymentRequestId in Metadata when creating invoice for Payment Request (#2644) @woutersamaey
+* Support multiple file upload (#2705) @cypherbeerus
```
Comment (NicolasDorier) [inline Changelog.md:24]:
```suggestion
* Support multiple file upload (#2705) @cypherbeerus
* Improve Dutch translation (https://github.com/btcpayserver/btcpayserver/commit/7ac83575d4c50e42f2ecc02c8bf80f66697b6d57)  @woutersamaey 
* Improve Portuguese translation (https://github.com/btcpayserver/btcpayserver/commit/7ac83575d4c50e42f2ecc02c8bf80f66697b6d57)  rafaelpac
```

**Comment by NicolasDorier** [comment]
This is fucking huge
image

**Comment by Zaxounette** [comment]
I can bet half the people reading this don't know what Astérix & Obélix is :D

File: Changelog.md [hunk 10]
```diff
@@ -1,5 +1,60 @@
...
+* Improve Dutch translation (https://github.com/btcpayserver/btcpayserver/commit/7ac83575d4c50e42f2ecc02c8bf80f66697b6d57)  @woutersamaey 
+* Improve Portuguese translation (https://github.com/btcpayserver/btcpayserver/commit/7ac83575d4c50e42f2ecc02c8bf80f66697b6d57)  rafaelpac
```
Comment (NicolasDorier) [inline Changelog.md:26]:
```suggestion
* Improve Portuguese translation (https://github.com/btcpayserver/btcpayserver/commit/7ac83575d4c50e42f2ecc02c8bf80f66697b6d57)  rafaelpac
* Improve payment view (#2748) @dennisreimann @dstrukt 
* Improve Wallet Send UI (#2750) @dennisreimann 
```

File: Changelog.md [hunk 11]
```diff
@@ -1,5 +1,60 @@
...
+* Fix issue with mysql migration and maxLength (#2541) @jkljajic
+* Fix broken shopify links @kukks
+* Fix bug with LN payment method API endpoint throwing 500 (#2567) @bolatovumar
+* Fix various wording and typos @pavlenex @britttttk @Zaxounette Jimi Ford
+* Fix visual bug with invoices search help text overlapping invoice action buttons (#2583) @bolatovumar
+* Fix: Invoice Search Text crashes invoice creation when value is too long (#2675) @kukks
+* Greenfield documentation fixes (#2657 #2674 #2681 #2598) @woutersamaey @bolatovumar
+* Re-enable "Create" button for invoices on correct form input (#2694) @bolatovumar
+* Fix: Payment Request status does not update on invoice marked events or when pr amount is changed (#2700) @kukks
+* Properly clip taxIncluded and invoice's amount (#2724) @nicolasdorier
```
Comment (NicolasDorier) [inline Changelog.md:56]:
```suggestion
* Properly clip taxIncluded and invoice's amount (#2724) @nicolasdorier
* Fix PoS bug on dark mode (#2743) @dennisreimann 
```

**Comment by NicolasDorier** [comment]
#2760

**Comment by NicolasDorier** [comment]
#2764

File: Changelog.md [hunk 12]
```diff
@@ -1,5 +1,63 @@
...
+* Hide referer URL to hide our BTCPay Server URL (#2655) @woutersamaey
+* Deeper accessibility for plugin system @kukks
+* Add webhook delivery status indicator (#2679) @bolatovumar
+* Auto-select store when creating a new invoice (#2680) @bolatovumar
+* Save paymentRequestId in Metadata when creating invoice for Payment Request (#2644) @woutersamaey
+* Support multiple file upload (#2705) @cypherbeerus
+* Improve Dutch translation (https://github.com/btcpayserver/btcpayserver/commit/7ac83575d4c50e42f2ecc02c8bf80f66697b6d57)  @woutersamaey 
+* Improve Portuguese translation (https://github.com/btcpayserver/btcpayserver/commit/7ac83575d4c50e42f2ecc02c8bf80f66697b6d57)  rafaelpac
+* Improve payment view (#2748) @dennisreimann @dstrukt 
+* Improve Wallet Send UI (#2750) @dennisreimann
```
Comment (NicolasDorier) [inline Changelog.md:28]:
```suggestion
* Improve Wallet Send UI (#2750) @dennisreimann 
* Show new store warning icon only if neither on-chain wallet nor LN is configured (#2760) @bolatovumar 
* Update successful refund message (#2764) @cypherbeerus
* Fix translation on finnish, bulgarian, Kazath (fa91174b1a310e46a37e1862f2b9c263f5e26408, 10e3595a829052573a9918eacafabc6d10e03ea6 965beebc6624906a1f3127623576088dee23e9bf) @NicolasDorier 
```

File: Changelog.md [hunk 13]
```diff
@@ -1,5 +1,63 @@
...
+* Fix broken shopify links @kukks
+* Fix bug with LN payment method API endpoint throwing 500 (#2567) @bolatovumar
+* Fix various wording and typos @pavlenex @britttttk @Zaxounette Jimi Ford
+* Fix visual bug with invoices search help text overlapping invoice action buttons (#2583) @bolatovumar
+* Fix: Invoice Search Text crashes invoice creation when value is too long (#2675) @kukks
+* Greenfield documentation fixes (#2657 #2674 #2681 #2598) @woutersamaey @bolatovumar
+* Re-enable "Create" button for invoices on correct form input (#2694) @bolatovumar
+* Fix: Payment Request status does not update on invoice marked events or when pr amount is changed (#2700) @kukks
+* Properly clip taxIncluded and invoice's amount (#2724) @nicolasdorier
+* Fix PoS bug on dark mode (#2743) @dennisreimann
```
Comment (NicolasDorier) [inline Changelog.md:59]:
```suggestion
* Fix PoS bug on dark mode (#2743) @dennisreimann 
* Remove support for payout to a Bitcoin Url (#2766) @NicolasDorier 
```

**Pull request merged**